rpcs3/Emu/CPU/CPUThread.h

```diff
@@ -22,6 +22,7 @@ enum class cpu_flag : u32
 	signal, // Thread received a signal (HLE)
 	memory, // Thread must unlock memory mutex
 	pending, // Thread has postponed work
+	pending_recheck, // Thread needs to recheck if there is pending work before ::pending removal
```

Comment:
What's the point of fixing race between breakpoint set and actual execution if latter can always proceed with certain chance?

Reply:
If before a breakpoint was added there were no breakpoints and the SPU thread is currently executing cpu_work(), without this flag it can disable pending flag for good without  being aware of the new breakpoint being added and cpu_work() will cease to be called.